src/Utils/Utils.php

```diff
@@ -43,6 +42,16 @@ public static function removeControlCharacters(?string $string): string
         return $result ?? $string;
     }
 
+    /**
+     * Filter names are not allowed to have special characters inside of them. If they do, you may not be able
+     * to properly select a filter in certain scenarios. This is only relevant for internal naming, which is
+     * used for the query parameters. The naming in the customer-login can differ though.
+     */
+    public static function cleanFilterName(string $filterName): string
+    {
+        return preg_replace('/[^äöü_A-Za-z0-9:-]/u', '', $filterName);
```

Comment:
We are allowing umlauts?

Reply:
Maybe reorder to `A-Za-z0-9:-_`?

Reply:
HTML Element ids allow umlauts. Reordering makes sense tough.